crates/bevy_window/src/event.rs

```diff
@@ -58,10 +58,21 @@ pub struct WindowCloseRequested {
 pub struct WindowClosed {
     pub id: WindowId,
 }
-/// An event that is sent whenenver the user's cursor moves.
+/// An event reporting that the mouse cursor has moved on a window.
```

Comment:
```suggestion
/// An event reporting that the mouse cursor has moved inside a window.
```
Not too sure if "on a window" or "inside a window" is a better wording for these

Reply:
The winit docs says “on the window”: https://docs.rs/winit/0.26.1/winit/event/enum.WindowEvent.html#variant.CursorMoved

Reply:
Awesome, then let's keep it as is